packages/edit-post/src/components/sidebar/settings-header/index.js

```diff
@@ -11,11 +12,12 @@ import SidebarHeader from '../sidebar-header';
 
 const SettingsHeader = ( { openDocumentSettings, openBlockSettings, sidebarName } ) => {
 	const blockLabel = __( 'Block' );
+	const documentLabel = applyFilters( 'editor.sidebar.settingsHeader.documentLabel', __( 'Document' ) );
 	const [ documentAriaLabel, documentActiveClass ] = sidebarName === 'edit-post/document' ?
 		// translators: ARIA label for the Document sidebar tab, selected.
-		[ __( 'Document (selected)' ), 'is-active' ] :
+		[ `${ documentLabel } ${ __( '(selected)' ) }`, 'is-active' ] :
 		// translators: ARIA label for the Document sidebar tab, not selected.
```

Comment:
This comment may be better situated above line 15.

Reply:
0de2bb4